src/components/badges/BadgeSection.tsx

```diff
@@ -30,10 +31,11 @@ export default function ({
   onMinted?: () => void
 }) {
   if (minted.length === 0 && proofs.length === 0) return null
+  const listRef = useAutoAnimate({})
```

Comment:
Also, is it possible to set it as `useAutoAnimate<HTMLDivElement>`?

Reply:
It doesnt eat typings